tests/test_auth.py

```diff
@@ -1,37 +1,39 @@
+from pathlib import Path
+
 import pytest
 
 from aocd.exceptions import AocdError
 from aocd.models import default_user
 
 
-def test_no_session_id(test_token, capsys):
+def test_no_session_id(test_token: Path, capsys: pytest.CaptureFixture[str]) -> None:
     test_token.unlink()
-    with pytest.raises(AocdError("Missing session ID")):
+    with pytest.raises(AocdError("Missing session ID")): # type: ignore[call-overload] # using pytest-raisin
```

Comment:
There are 41 occurences of this pragma added 

    # type: ignore[call-overload] # using pytest-raisin

Is there a way to just globally disable the type of overload which pytest-raisin in mypy configuration or something? It adds a lot of noise